server/src/main/java/com/project/cisco/controler/MessageApiController.java

```diff
@@ -0,0 +1,49 @@
+package com.project.cisco.controler;
+
+import com.project.cisco.api.MessageApi;
+import com.project.cisco.dto.MessageDto;
+import com.project.cisco.service.MessageService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
```

Comment:
Let's add here origin to fix an error releated to CORS on frontend

`@CrossOrigin(value = { "http://localhost:3000/" },
        maxAge = 900
)`

Reply:
done